Zend/tests/enum/no-unset-propertes.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Enum properties cannot be unset
+--FILE--
+<?php
+
+enum Foo {
+    case Bar;
+}
+
+$foo = Foo::Bar;
+
+try {
+    unset($foo->case);
+} catch (Error $e) {
+    echo $e->getMessage() . "\n";
+}
+
+?>
+--EXPECT--
+Enum properties are immutable
```

Comment:
nit: Would be useful to test that cases can be read from and have the expected value (Bar?) - the only 2 tests involving the property is a test of their immutability

nit: Would it be useful to mention more details such as "Saw attempt to modify property of enum case Foo::Bar - enum properties are immutable" (or "Cannot modify Foo::Bar: enum properties are immutable")  to make it easier for users to debug this exception if they saw it in an application

Reply:
All error messages will need some work, I'm closing this for now.